src/resourcemgr/tpl/fileSelect.tpl

```diff
@@ -1,22 +1,26 @@
 {{#each files}}
-<li data-type="{{type}}" data-file="{{uri}}" data-display="{{display}}" data-mime="{{mime}}" data-size="{{size}}" data-url="{{viewUrl}}" data-download="{{downloadUrl}}" data-alt="{{alt}}">
-    <span class="desc truncate">{{name}}</span>
-    <div class="actions">
-        <div class="tlb">
-            <div class="tlb-top">
-                <span class="tlb-box">
-                    <span class="tlb-bar">
-                        <span class="tlb-start"></span>
-                        <span class="tlb-group">
-                            <a href="#" class="tlb-button-off select" title="{{__ 'Select this file'}}"><span class="icon-move-item"></span></a>
-                            <a href="{{downloadUrl}}" download="{{name}}" target="_blank" class="tlb-button-off download" title="{{__ 'Download this file'}}"><span class="icon-download"></span></a>
-                            <a href="#" class="tlb-button-off" title="{{__ 'Remove this file'}}" data-delete=":parent li"><span class="icon-bin"></span></a>
+    <li data-type="{{type}}" data-file="{{uri}}" data-display="{{display}}" data-mime="{{mime}}" data-size="{{size}}" data-url="{{viewUrl}}" data-download="{{downloadUrl}}" data-alt="{{alt}}">
+        <span class="desc truncate">{{name}}</span>
+        <div class="actions">
+            <div class="tlb">
+                <div class="tlb-top">
+                    <span class="tlb-box">
+                        <span class="tlb-bar">
+                            <span class="tlb-start"></span>
+                            {{#if permissions.read }}
+                                <span class="tlb-group">
+                                    <a href="#" class="tlb-button-off select" title="{{__ 'Select this file'}}"><span class="icon-move-item"></span></a>
```

Comment:
This should be a button instead of a link. Considering that merging is urgent, we can postpone the fix and add it to a11y improvements after training.

Reply:
I think is better to postpone taking a look at pressure to deliver this